Jenkinsfile

```diff
@@ -1,10 +1,18 @@
 // Obtain files from source control system.
 if (utils.scm_checkout()) return
 
+withCredentials([string(
+    credentialsId: 'drizzlepac-codecov',
+    variable: 'codecov_token')]) {
 // Generate installation compatibility matrix
+/*
 matrix_python = ["3.6", "3.7"]
 matrix_astropy = [">=3.1.0"]
-matrix_numpy = [">=1.14", "==1.15.0"]
+matrix_numpy = ["<1.15", "<1.16"]
+*/
+matrix_python = ["3.7"]
```

Comment:
Was the 3.6 entry here removed on purpose?

Reply:
Negative, I'll change it back. I was only avoiding the long wait (and/or halt-and-catch-fire)